internal/handlers/shared/msg_buildinfo.go

```diff
@@ -47,3 +49,11 @@ func (h *Handler) MsgBuildInfo(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 
 	return &reply, nil
 }
+
+func buildEnvironment() (buildEnvironment types.Document) {
```

Comment:
I think we should not extract that function – we use it only in one place

Reply:
Will fix. Also, maybe it would be better to initialize the `buildEnvironment` document during the start, kinda like `version.info` is initialized?
If so, is it ok if I put `buildEnvironment` in the `version` package or should it reside elsewhere?

Reply:
Btw, we can initialize the `buildEnvironment` Document during `version.init()`, so no map iteration and copying during runtime. Maybe do that?

Reply:
Sure, let's do that